tests/test_routes.py

```diff
@@ -55,21 +73,81 @@ def test_index(self):
         resp = self.client.get("/")
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
 
+    def test_create_pet(self):
```

Comment:
**Nit:** found a few instances with "create pet" instead of "create product"

Reply:
Will edit it and push